usr/local/www/diag_resetstate.php

```diff
@@ -43,21 +43,31 @@
 ##|*MATCH=diag_resetstate.php*
 ##|-PRIV
 
+define("DEBUG", true);	// Displays "success" messages, but does not call the functions that reset the firewall state tables
```

Comment:
I think the issue I have with it is that everyone will have other needs when debugging. You're 'claiming' a generic term here, but you are using it for something specific (it's not debugging; you're just making the back-end readonly).

If you fix my 2 comments I'm ready to merge, could you also remove the DEBUG for now? :)